### DISCUSSION TITLE
Deleted

### DISCUSSION THREAD

**Discussion by monalos** [cedricp/ddt4all#919] [General]
.

**Comment by AndyIreland** [comment]
try different scenarios - try increasing 'CAN TIMEOUT' to 600ms.

Try programming with engine running.
Try programming with engine off.
Try programming with engine off but keycard in slot.
Try programming with engine off but a long press on the start button until 'remove keycard' comes up on dash or 'ACC' - but leave keycard in.
Try programming with engine off but gear selector in 1st gear (or Drive if auto gearbox)  

try getting a more compatible ELM327 (one preferably with the correct PIC chip in it) 

Hope this helps.

**Reply by monalos** [reply]
Thanks 

I can code everything else, it only gives this error in res_ start coding. I will try what you said

ELM THİS photo

![IMG_6723](https://github.com/cedricp/ddt4all/assets/76909804/70cd32f1-2b34-45f0-a7a5-3804497468e1)

**Comment by matzel82** [comment]
How want you start remote your engine ?

**Reply by monalos** [reply]
with key card

**Reply by matzel82** [reply]
Sorry...this won't work

**Reply by monalos** [reply]
Why?

**Reply by matzel82** [reply]
There isn't a button to do this... I haven't seen a car with this...  it's doesn't mean that you only with "coding" get this feature. You have to reprogram any ECUS. And without unlocking the Mobilizer you won't start the engine. Second you have switch APC+ to some ECUs.

**Reply by monalos** [reply]
Renault arkana , captur , talisman , dacia duster , 

These cars also have remote start. There is a 2s button on the remote control.

**Reply by monalos** [reply]
I'm not asking if this is possible right now, I'm investigating the reason why elm gives an error in ddt4all.

**Comment by AndyIreland** [comment]
What infotainment have you got on your Clio4? 

If you have medianav you more than likely have to enable RES using MICOM Manager and if its a Rlink1 fitted to your Clio4 you might have to enable the setting in that too

Micom Test manager setting in Clio 4 Medianav Second edition:
![image](https://github.com/cedricp/ddt4all/assets/49172217/bd1f24b5-17cc-4ad6-b46d-9028c3ed303a)

**Reply by monalos** [reply]
Medinav was removed, android multimedia was installed. Is there any chance of activating it without Medineva? If it doesn't activate without Medinav, I can turn Medinav back on.

**Reply by monalos** [reply]
I enabled this setting when I first purchased Elm327. Then I closed it and when I tried to open it again, it gave the error in the first picture. Could it be from elm?

**Comment by AndyIreland** [comment]
it sounds like the cars with RES feature are cars produced in 'Cold climate countries' and is extra module fitted to those cars so maybe this why it wont enable? - just a thought. 

https://lamiaclio-forumcommunity-net.translate.goog/?t=61123693&_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=wapp

**Reply by monalos** [reply]
Automatic operation is done through Medinav. It works automatically when it reaches the temperature you set. I'm researching how to operate it from the remote control. In cold countries, there is both automatic operation and remote control. In fact, in an article I read, it can only operate from the remote control, without Medinav.

**Comment by AndyIreland** [comment]
Renault remote control card : Starting the engine remotely:
https://youtu.be/iVOtOyyyuXs?si=AD70fiPlct9bMVEE

**Reply by monalos** [reply]
Thanks

**Reply by peace1313** [reply]
Were you able to start the engine remotely? I activated it, the calendar appeared but the feature did not work.

**Reply by monalos** [reply]
What is the uch/bcm and hfm version in the car? How did you become active?

**Comment by peace1313** [comment]
Were you able to start the engine remotely? I activated it, the calendar appeared but the feature did not work.

**Reply by monalos** [reply]
No ı dont have remote start because remote start no key

**Reply by monalos** [reply]
What is the uch/bcm and hfm version in the car? How did you become active?

**Comment by evgenybozhko2** [comment]
Haha, this feature can be activated only on CLEAN UCH ecu.

**Reply by evgenybozhko2** [reply]
I'm about UCH/BCM module. Only on clean module, this feature can be activated. But completely works feature, you should get only in case if your car has a HFM module, and EMS module of engine. Or some other ECM with a dual immobilizer module. So if you don't know what is that, my help for you, just forgot about that.

**Reply by monalos** [reply]
I have the latest version keyless enabled HFM module in my car.

**Reply by evgenybozhko2** [reply]
Does ECM contains a dual immo ?

**Reply by monalos** [reply]
> ECM'de çift immo var mı?

I dont know , I don't know, I think not, I need to find an ECM version with dual immo.

**Reply by monalos** [reply]
Where can I check if there is dual immo?

**Reply by evgenybozhko2** [reply]
You should clear your bcm with renolink. Make it like a new one. Turn on feature remote engine start in bcm. Than pair bcm into car again via dealer can clip or online access. 

about ecm, also you should check that possible does Immo2 was respond with can bus frames. For example with same renolink or ddt4all with your ecm layout.

**Reply by monalos** [reply]
I understand, thank you, if there is no immo2 in the ecm, is it correct that I change it?

**Reply by evgenybozhko2** [reply]
Yes. But I'm not sure where you get it.

**Reply by monalos** [reply]
Have you done it before? Does your car now start remotely? It wouldn't bother me much if you share the part codes.

**Reply by evgenybozhko2** [reply]
Nope, now it blinking three times because of an error and stop the process or car run before starter works. Diesel ecm does not have a dual immo.. my is SID306 so sad.

**Reply by evgenybozhko2** [reply]
Also, electrical scheme is different, you should change it as well modules.

**Reply by monalos** [reply]
did you change the remote? Did you get the key with the 2s button on it?

**Reply by evgenybozhko2** [reply]
> did you change the remote? Did you get the key with the 2s button on it?

ofcourse

**Reply by monalos** [reply]
Which country are you in? There is no remote control in my country? Do you have whatsapp or telegram? Can I add you? we exchange ideas together

**Reply by evgenybozhko2** [reply]
> Which country are you in? There is no remote control in my country? Do you have whatsapp or telegram? Can I add you? we exchange ideas together

Ukraine. And yes, no remotes. All changes I was made by self.

**Reply by evgenybozhko2** [reply]
How I can find you?

**Reply by evgenybozhko2** [reply]
![Screenshot 2024-07-23 at 15 44 27](https://github.com/user-attachments/assets/93a42241-e8c5-4dca-b811-6da93334bb90)

**Reply by monalos** [reply]
This is Whatsapp number

**Reply by estebanfree** [reply]
> This is Whatsapp number

helo, Could you help me about RES project? thanks

**Reply by estebanfree** [reply]
> How I can find you?

hello, could you help me about RES proejct? thanks

**Reply by monalos** [reply]
> > How I can find you?
>
> hello, could you help me about RES proejct? thanks

Leave me a WhatsApp number and I'll find you.

**Reply by estebanfree** [reply]
> > > How I can find you?
> > 
> > 
> > hello, could you help me about RES proejct? thanks
> 
> Leave me a WhatsApp number and I'll find you.
thanks +905305652306

**Comment by AndyIreland** [comment]
I have seen on one ECU (more than likely the keycard programming) in DDT4ALL settings something to say What keycard , or how many keys, are on the keycard - you might have to change that setting too , to get RES working

**Reply by monalos** [reply]
> Bir ECU'da (büyük ihtimalle anahtar kartı programlamasında) DDT4ALL ayarlarında anahtar kartında hangi anahtar kartının veya kaç anahtarın olduğunu söyleyen bir şey gördüm - RES'in çalışmasını sağlamak için bu ayarı da değiştirmeniz gerekebilir

Yes, I also saw a setting that says which of the 4 added keys should do the remote start.
